service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterComponentsMapperTest.java

```diff
@@ -405,6 +405,25 @@ public void When_MappingWithRelatedProblemWithIncorrectProblemExtensionUrl_Expec
             .isEqualToIgnoringWhitespace(expectedXml);
     }
 
+    @Test
+    public void When_MappingWithRelatedProblemWithIncorrectProblemExtensionExtensionUrl_Expect_UnspecifiedProblemWithOriginalText() {
```

Comment:
```suggestion
    public void When_MappingWithRelatedProblemWithIncorrectProblemExtensionExtensionUrl_Expect_UnspecifiedProblemWithOriginalText() {
```

When mapping what? 😄